protocol/contracts/amo/AuraStaking.sol

```diff
@@ -45,10 +45,10 @@ contract AuraStaking is IAuraStaking, TempleElevatedAccess {
         emit SetAuraPoolInfo(_pId, _token, _rewards);
     }
 
```

Comment:
FYI a couple more typos/links were updated for issue https://github.com/yAudit/temple-lending-report/issues/5 in https://github.com/TempleDAO/temple/pull/842/commits/a5da3d96afdac0632a04aa73aa05834e1182e48c